docs/SSO.rst

```diff
@@ -1717,22 +1717,15 @@ home folders of SP and IDP.
 eXo Platform can perform both roles SP and IDP; thus, it can integrate 
 with services like Salesforce and Google Apps.
 
-.. note:: -  SAML2 IDP integration only works for eXo Platform on Jboss.
-
-          -  SAML2 SP integration works for eXo Platform Jboss and Tomcat.
+.. note::  -  SAML2 SP integration works for eXo Platform Tomcat.
+           -  SAML2 IDP integration is more supported.
```

Comment:
to delete all this **note**

> .. note::  -  SAML2 SP integration works for eXo Platform Tomcat.
>            -  SAML2 IDP integration is more supported.

(FYI:
> is more supported
=>
> is no more supported

Reply:
yep it was a typing error thanks

Reply:
Since we show the SAML workflows with SP and IDP roles identified in a diagram above, I think a note should exist but be more explicit and say : 

eXo Platform SAML integration supports the SP role thus can be integrated with various IdP providers (https://en.wikipedia.org/wiki/SAML-based_products_and_services) such as Salesforce or Shibboleth.

(and I think we should have a tutorial with Shibboleth in addition to salesforce as it seems popular for on premise scenarios.)